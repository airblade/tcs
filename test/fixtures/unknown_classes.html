<div class="p-3 shadow-xl foo">
</div>
<div class="p-3 shadow-xl bar">
</div>
