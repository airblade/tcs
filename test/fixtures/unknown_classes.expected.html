<div class="foo p-3 shadow-xl">
</div>
<div class="bar p-3 shadow-xl">
</div>
